docs/source/examples/demo_deploy_cliptextmodel.ipynb

```diff
@@ -0,0 +1,343 @@
+    "text_to_encode = \"example search query\" #See https://huggingface.co/docs/transformers/torchscript for more info on dummy inputs\n",
+    "\n",
+    "# Instantiate CLIPTextModel and CLIPProcessor with pretrained weights\n",
+    "model = CLIPTextModel.from_pretrained(model_name, torchscript=True, return_dict=False)\n",
+    "processor = CLIPProcessor.from_pretrained(model_name)\n",
+    "\n",
+    "# Use processor to generate tensors and create dummy input\n",
+    "text_inputs =processor(text=text_to_encode, return_tensors=\"pt\",max_length=77, padding=\"max_length\", truncation=True)\n",
+    "dummy_input = [text_inputs['input_ids'], text_inputs['attention_mask']]\n",
+    "\n",
+    "# Trace model and convert to torchscript object\n",
+    "traced_model = torch.jit.trace(model, dummy_input)\n",
+    "\n",
+    "# Save model in portable format\n",
+    "torch.jit.save(traced_model, \"traced_model_example.pt\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Prep files for registration\n",
+    "\n",
+    "OpenSearch requires two files zipped together for registration:\n",
+    "* Model in TorchScript format\n",
+    "* tokenizor.json file\n",
+    "\n",
+    "The tokenizor for the model used in this example can be found [here](https://huggingface.co/openai/clip-vit-base-patch32/tree/main)\n",
+    "\n",
+    "Additionally, a config.json file with the following details must be passed with the .zip. More info on [model config](clip-vit-base-patch32) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'\\n{\\n    \"name\": \"clip-vit-base-patch32\",\\n    \"version\": \\'1.0.0\\',\\n    \"model_format\": \"TORCH_SCRIPT\",\\n    \"model_config\": {\\n        \"model_type\": \"clip\",\\n        \"embedding_dimension\": 512,\\n        \"framework_type\": \"huggingface_transformers\"\\n    }\\n}\\n'"
+      ]
+     },
+     "execution_count": 57,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# config.json sample contents\n",
+    "\"\"\"\n",
+    "{\n",
+    "    \"name\": \"clip-vit-base-patch32\",\n",
+    "    \"version\": '1.0.0',\n",
+    "    \"model_format\": \"TORCH_SCRIPT\",\n",
+    "    \"model_config\": {\n",
+    "        \"model_type\": \"clip\",\n",
+    "        \"embedding_dimension\": 512,\n",
+    "        \"framework_type\": \"huggingface_transformers\"\n",
+    "    }\n",
+    "}\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3: Register model to OpenSearch\n",
+    "\n",
+    "* Model name in config.json should match .pt torchscript file name\n",
+    "* Record the model ID from the output of the next cell\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
```

Comment:
Looks like there's an error?

Reply:
Accidentally ran this cell before loading the packages which caused this error. It should not be part of the commit - the notebook runs if executed in sequential order